### DISCUSSION TITLE
Fingerprint seems to be unique

### DISCUSSION THREAD

**Discussion by hkaancaliskan** [bromite/bromite#1670] [Q&A] [answered]
I checked bromite fingerprint uniqueness through EFF and BrowserLeaks and both says it's unique.
![Screenshot_20211219-160417_Bromite](https://user-images.githubusercontent.com/17825767/146675865-4f7a4677-1d6b-421c-bc76-627a284a5a89.jpg)
![Screenshot_20211219-160335_Bromite](https://user-images.githubusercontent.com/17825767/146675866-f7dd9b13-5293-4b2e-bc7f-9b0d59edc385.jpg)

My bromite settings are mostly default. I can provide the changes i made if necessary. I didn't change UA.
I'm using Cloudflare dns through Android 11 system settings.

Is this just for me or for everyone?

https://browserleaks.com/canvas
https://coveryourtracks.eff.org/

**Comment by peternrdstrm** [comment, marked as answer]
Yes, it is true that the fingerprints are unique. What you missed, is that they change, every time you refresh the site, meaning, that even if it's unique, it cannot be tied to you.

**Reply by hkaancaliskan** [reply]
Yep, that's changing, thank you
